### PR TITLE
Fix a couple of issues with multi-file loading of modules

### PR DESCRIPTION
**Pull request by applmak**

### CODE REVIEW

**Comment by applmak** [comment]
This should fer-realz fix #93.

File: lib/fake_require.js [hunk 1]
```diff
@@ -52,8 +52,14 @@ if (typeof define === 'function' && define.amd) {
           // First, create a custom requirejs context. Note that we use the
           // global requirejs, and not the 'local' require, as they are, in 
           // fact, different functions with different kinds of functionality.
-          let contextRequire = requirejs.config({context: name});
-        
+          let config = {context: name};
```
Comment (iatkinson) [inline lib/fake_require.js]: want to just do this all in one statement:

```
const config = {
  context: name,
  paths: ...
};
```
?

File: lib/fake_require.js [hunk 2]
```diff
@@ -52,8 +52,14 @@ if (typeof define === 'function' && define.amd) {
...
+          // Copy over the paths from the base context so that library deps
+          // like 'three' or 'p5' work correctly.
+          config.paths = _.clone(requirejs.s.contexts._.config.paths);
+          // Name the context.
+          config.context = name;
```
Comment (iatkinson) [inline lib/fake_require.js]: redundant

File: lib/fake_require.js (hunk 1 above)
Comment (applmak) [inline lib/fake_require.js]: Done.

File: lib/fake_require.js (hunk 2 above)
Comment (applmak) [inline lib/fake_require.js]: Durr.

**Pull request merged**